.gitignore

```diff
@@ -0,0 +1,120 @@
+# Logs
```

Comment:
Looks like you took someone's .gitignore file. You don't have all these extensions in your project. Please remove unnecessary types.

Reply:
Yeah, it's from here: https://github.com/github/gitignore/blob/master/Node.gitignore
I agree, in this project it is necessary to exclude only node_modules/ and dist/